acme/setup.py

```diff
@@ -7,7 +7,9 @@
 
 install_requires = [
     'cryptography>=3.2.1',
-    'josepy>=1.13.0',
+    # Josepy 2+ introduce backward incompatible changes by droping usage of
+    # deprecated PyOpenSSL APIs.
```

Comment:
we and especially @wgreenberg have actually been talking a bit about tackling https://github.com/certbot/certbot/issues/8322 instead of updating josepy. i'm fine with pinning back `<2` to be safe, but let's soften this language a bit while we're still deciding for sure what we're gonna do

```suggestion
    # Josepy 2+ may introduce backward incompatible changes by droping usage of
    # deprecated PyOpenSSL APIs.
```